packages/components/organisms/f-checkout/src/store/checkout.module.js

```diff
@@ -356,9 +356,19 @@ export default {
 
             if (isAddressInLocalStorage) {
                 const storedAddress = addressService.getAddressFromLocalStorage(false);
+
                 if (addressService.doesAddressInStorageAndFormMatch(storedAddress, state.address)) {
                     addressCoords = [storedAddress.Field2, storedAddress.Field1];
                     commit(UPDATE_GEO_LOCATION, addressCoords);
+                } else if (window.localStorage) {
+                    const addressDetails = state.address;
+
+                    window.localStorage.setItem('je-full-address-details', JSON.stringify({
+                        PostalCode: addressDetails.postcode,
+                        Line1: addressDetails.line1,
```

Comment:
Do we not also need to add the new coordinates we retrieve from the API into Field1 and Field 2 here as well?

Reply:
The coordinates come from Loqate, which is the call from the searchbox on home so we won't be able to update that at checkout right now. If the user completes payment then goes back to home for another order in the future and searches then the lat long gets pulled through. For it to work in checkout we need  a Loqate call to be added.

Reply:
Oh okay, makes sense but then won't the stored address and stored coordinates mismatch? Because the coordinates we'll have will be for the previous address?

Reply:
Should clear it and only add the edited values to local storage.. Or have i missed anything?